dot/core/service.go

```diff
@@ -34,7 +34,7 @@ var (
 // QueryKeyValueChanges represents the key-value data inside a block storage
 type QueryKeyValueChanges map[string]string
 
-type wasmerInstanceFunc func(code []byte, cfg *wasmer.Config) (instance *wasmer.Instance, err error)
+type wasmerInstanceFunc func(code []byte, cfg runtime.InstanceConfig) (instance *wasmer.Instance, err error)
```

Comment:
No longer a pointer?

Reply:
Saw the other changes, answered my own question

Reply:
Nah just a dumb struct. No need for a pointer, the struct isn't that big and contains slices/maps which are themselves pointers. FYI and counter-intuitively, most of the time in Go, it's faster to copy structs on the stack than passing a pointer (which also involves load on the GC).